.github/actions/install-parcels/action.yml

```diff
@@ -18,14 +15,17 @@ runs:
       uses: al-cheb/configure-pagefile-action@v1.3
       with:
         minimum-size: 8GB
-    - name: Install micromamba (${{ inputs.environment-file }})
-      uses: mamba-org/setup-micromamba@v1
+    - name: Install miniconda (${{ inputs.environment-file }})
+      uses: conda-incubator/setup-miniconda@v3
```

Comment:
> is there no way to always use the latest version?

only if they have a `latest` tag in the repo itself. Most of the time when using actions there is a major version specified so that a new version (with a different API) doesn't break existing uses.

Note that this has no bearing on the version of conda installed, just the version of the action itself (as the conda version is handled separately through a parameter).